pkg/internal/export/otel/metrics.go

```diff
@@ -41,6 +42,10 @@ const (
 	SpanMetricsCalls      = "traces_spanmetrics_calls_total"
 	SpanMetricsSizes      = "traces_spanmetrics_size_total"
 	TracesTargetInfo      = "traces_target_info"
```

Comment:
From what I understood, Tempo generates `traces_target_info` when the OTel Collector Service Graph Connector generates `target_info` and we have a configuration flag to choose the one used (screenshot below).

@domasx2 do we have best practices here?

<img width="645" alt="image" src="https://github.com/grafana/beyla/assets/459691/e15cfb20-7e8e-4f39-be1e-04a38aa9511b">

Reply:
Does it mean we may not need it? I simply went by what I saw Tempo generated for my cloud instance when using Alloy.

Reply:
It's needed for sure. I think what @cyrille-leclerc wondering is wether you should align with Tempo metric generator metric names, or OTEL spanmetrics/ service graph metrics connector metric names. I don't have  a strong opinion TBH, though would lean towards OTEL

Reply:
Oh absolutely OTel then, just confirming, so if we renamed the collection as `target_info` App O11y will continue to work, correct? I'll create a PR to fix this.

Reply:
Note that span metric names change as well for Otel, instead of `traces_span_*` it's `duration_seconds_bucket`, `duration_seconds_count` and `duration_seconds_total`

Reply:
as @domasx2 said, the question of OTel vs Tempo naming conventions. I would see value in adopting OTel conventions.
That being said, what's our recommendation if a customer is getting started combining Beyla with a bunch of app instrumented with OTel SDKs the simplest possible way, relying on Tempo metrics generation?
Sorry it's more complicated than we would like.

Reply:
I do agree with the Gang that it would be beneficial to run with the new otel default. Given that users can have more instrumentation than just a single beyla deployment, I would opt to make this configurable to be either Otel (default) OR tempo.

If it was flexible, people could rely on tempo generated metrics + the ones from Beyla.

I would also lean towards helping tempo to align with otel long-term.

Reply:
Alright, I think we have a consensus. We'll add an OTel format and make it default.

I opened a new issue to track this here https://github.com/grafana/beyla/issues/821